src/dotnet/CommandDirectoryContext.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.IO;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal static class CommandDirectoryContext
```

Comment:
There exists an overload of `Path.GetFullPath` that [takes a base](https://github.com/dotnet/coreclr/pull/15579) other than CWD, if desired.  Could that be leveraged instead of this type?

Reply:
Hah that one is so new I didn't know about it 😂 
I'll change it.